pkg/rpcclient/gas/gas.go

```diff
@@ -0,0 +1,26 @@
+/*
+Package gas provides a convenience wrapper for GAS contract to use it via RPC.
+
+GAS itself only has standard NEP-17 methods, so this package only contains its
+hash and allows to create NEP-17 structures in an easier way.
+*/
+package gas
```

Comment:
BTW, I'm also thinking about `gastoken` and `neotoken`.

Reply:
I'm OK with just `gas` and `neo`, because we already have an external interop API with `gas` and `neo` packages. With it our native interop wrappers and RPC client interface look unified. Also, `gas` and `neo` are shorter, which is nice IMO.

Reply:
Do you want to rename them because of such situations?
```
gasprom := gas.New(act)
```

Reply:
Yeah. We use `gas` a lot for variable names. But some symmetry with interops is also a valid argument.

Reply:
We may leave it to the user side, in case if he needed he can always use an import alias. But generally, I'm not against renaming.